src/mcc_base.jl

```diff
@@ -156,9 +156,9 @@ For each clade `m` in `MCC`:
   is an integer starting at `label_init`.
 - Rename each non-leaf internal node of `m` to `shared_\$i_\$j` where `j` is an index
   specific to `m`.
-
+If given as a parameter update the `shared_maps` dictionary with the new node label names
```

Comment:
Not sure what `shared_maps` is supposed to be, a quick example in the docstring?